src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+

```

Comment:
```suggestion

```

Reply:
Methods declarations must be separated from the previous code with an empty line